src/discussions/comments/CommentsView.jsx

```diff
@@ -41,13 +42,16 @@ function CommentsView({ intl }) {
   return (
     <div className="discussion-comments d-flex flex-column w-100 ml-3">
       <div className="mb-2">
+        <PostBanner post={thread} intl={intl} />
         <div className="list-group list-group-flush">
           <Post post={thread} />
           <div className="list-group">
             {comments.map(reply => (
-              <div key={reply.id} className="list-group-item list-group-item-action">
-                <Reply reply={reply} />
-              </div>
+              <><PostBanner post={reply} intl={intl} />
+                <div key={reply.id} className="list-group-item list-group-item-action">
+                  <Reply reply={reply} />
+                </div>
+              </>
```

Comment:
Putting the banner like this means you need to move the key to the empty fragment.